pkg/templates/loader/testdata/with_output/expected.yaml

```diff
@@ -13,6 +13,7 @@ openAPISchema:
             x-walrus-ui:
               colSpan: 12
               order: 1
+              widget: YamlEditor
```

Comment:
What if we don't set widget in schema? Will UI fall back to use the yaml editor?